test/plugins/tokens.test.js

```diff
@@ -141,7 +144,7 @@ describe('token plugin test', () => {
         });
 
         it('returns 201 and correct token data', () => {
-            tokenMock = getTokenMock(Object.assign({}, testToken, { value: '1234' }));
+            tokenMock = getTokenMock(Object.assign({}, testToken, { value: testValue }));
```

Comment:
Is it possible to just use `testTokenWithValue` here or do `getTokenMock(Object.assign({}, testTokenWithValue)` ?